spec/helpers/case_workers/travel_helper_spec.rb

```diff
@@ -7,9 +7,9 @@
     let(:expense) { create(:expense, :with_calculated_distance, mileage_rate_id: mileage_rate, location: 'Basildon', date: 3.days.ago) }
 
     subject(:link) do
-      Nokogiri::HTML.fragment(helper.link_to_map(expense, origin: 'SE9 2XX', destination: 'SE9 3XX', target: '_blank'))
-                    .search('a')
-                    .first
+      Nokogiri::HTML4.fragment(helper.link_to_map(expense, origin: 'SE9 2XX', destination: 'SE9 3XX', target: '_blank'))
```

Comment:
all fine but
> Nokogiri::HTML has been renamed to Nokogiri::HTML4, and Nokogiri::HTML is aliased to preserve backwards-compatibility.

Reply:
> Future releases of Nokogiri may deprecate HTML methods or otherwise change this behavior, so please start using HTML4 in place of HTML.

Do you think it should be changed to something else?